engine/apps/api/tests/test_alert_receive_channel.py

```diff
@@ -57,6 +57,28 @@ def test_get_alert_receive_channel_by_integration_ne(
         assert result["integration"] != AlertReceiveChannel.INTEGRATION_DIRECT_PAGING
 
 
+@pytest.mark.django_db
+def test_get_alert_receive_channel_by_id_ne(
+    make_organization_and_user_with_plugin_token, make_user_auth_headers, make_alert_receive_channel
+):
+    organization, user, token = make_organization_and_user_with_plugin_token()
+
+    alert_receive_channel_1 = make_alert_receive_channel(organization)
+    alert_receive_channel_2 = make_alert_receive_channel(organization)
+    alert_receive_channel_3 = make_alert_receive_channel(organization)
```

Comment:
maybe add some alert_receive_channel in a different organization to the test so we are sure we not include things we shouldn't?

Reply:
good call, added in https://github.com/grafana/oncall/pull/4032/commits/eb3bb23d70716c4f4327d46750680c74b9f8b306